dashboard/app/views/levels/editors/_frequency_analysis.html.haml

```diff
@@ -1,4 +1,8 @@
 .field
+  = f.label :cipher
+  = f.select :cipher, options_for_select(@level.class.ciphers, @level.cipher)
```

Comment:
Worth a little inline documentation here describing the difference between the modes?